src/app/material-timepicker/ngx-material-timepicker.component.ts

```diff
@@ -73,14 +73,14 @@ export class NgxMaterialTimepickerComponent implements TimepickerRef {
 
     @Input()
     set filter(
-        fn: (time: DateTime, granularity?: "hours" | "minutes") => boolean
+        fn: (time: DateTime, granularity?: 'hours' | 'minutes') => boolean
     ) {
         this._filter = fn;
```

Comment:
why do you need setter and getter? Use just `@Input() filter: type`

Reply:
changed in latest commits.